_posts/2024-02-22-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  "Enhancing Security in OpenSearch 2.12.0: The End of Default Admin Password"
+authors:
+ - dchanp
+date: 2024-02-22
+categories:
+ - community-updates
+ - technical-posts
+meta_keywords: "OpenSearch Security, Security Demo Configuration"
+meta_description: Learn about the changes to Security plugin's demo configuration setup
+has_math: false
+has_science_table: false
+---
+
+
+In the landscape of data management and search technologies, OpenSearch stands out by offering a comprehensive set of features and a flexible platform for users globally. With the rollout of OpenSearch version 2.12.0, an update aimed at improving the initial setup process for "admin" user using the Security plugin's demo configuration tool has been introduced.
+
+**Rethinking Default Admin Credentials**
+
+Until the release of OpenSearch 2.11.1, a default 'admin' password was included in the demo configuration file of the Security plugin, intended to streamline the setup for new users. While this approach was convenient, it also posed a potential security risk. Despite numerous warnings in the demo tool, documentation, and console outputs about moving demo setups to production with the default password, occurrences of this issue persisted, leading to the decision to eliminate the default password for the admin user, who has comprehensive access, and can leave the cluster vulnerable if the default password is not updated.
```

Comment:
In the last sentence, does "can" refer to "user"? If so, the last comma should be removed.